app/assets/stylesheets/dlme.scss

```diff
@@ -19,3 +19,7 @@
   width: 80%;
   margin-left: 10%;
 }
+
+caption {
+  color: #333;
```

Comment:
We probably want to change `text-mute` globally, yeah? And, let's figure out why a stand-alone variables file isn't working, if that's the blocker.